connectors/sources/tests/test_google_cloud_storage.py

```diff
@@ -32,6 +32,7 @@ def get_mocked_source_object():
         {"service_account_credentials": SERVICE_ACCOUNT_CREDENTIALS, "retry_count": 0}
     )
     mocked_gcs_object = GoogleCloudStorageDataSource(configuration=configuration)
+    mocked_gcs_object.get_storage_client()
```

Comment:
Do we need that here? This is probably here for the initialization side effect, right?
We may want a test for `get_storage_client`, that we get the same instance back on subsequent calls on the same data source instance?